MonadicBits/Either.cs

```diff
@@ -1,9 +1,19 @@
 ﻿using System;
 using System.Diagnostics.CodeAnalysis;
-using static MonadicBits.Functional;
+using MonadicBits.Either;
 
 namespace MonadicBits
 {
+    using static Functional;
+
+    public partial class Functional
+    {
+        public static Left<TLeft> Left<TLeft>(this TLeft value) => value;
+        public static Right<TRight> Right<TRight>(this TRight value) => value;
+
```

Comment:
I know it's not important, but because there is another thing to fix, could you be so kind and remove these two empty lines? ;)